examples/platform/silabs/SiWx917/SiWx917/sl_wlan_config.h

```diff
@@ -48,21 +48,11 @@ static const sl_wifi_device_configuration_t config = {
 #endif
                                                 | SL_SI91X_TCP_IP_FEAT_ICMP | SL_SI91X_TCP_IP_FEAT_EXTENSION_VALID),
                      .custom_feature_bit_map     = (SL_SI91X_CUSTOM_FEAT_EXTENTION_VALID | RSI_CUSTOM_FEATURE_BIT_MAP),
-                     .ext_custom_feature_bit_map = (
-#ifdef SLI_SI917
```

Comment:
Historically this define was use for the efr32+917 ncp combo to differentiate with the 9116. Are you sure this header is not included in a 9116 ncp build ? if so this could cause breakage.

Reply:
`#include "sl_wlan_config.h"` Is only present in `examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c`, which is only in the sources of the SiWx917 BUILD.gn. 

There is another `sl_wifi_if.c` under `examples/platform/silabs/rs911x/SiWx917/sl_wifi_if.c`, so I suspect it was copied over.

Reply:
efr32/rs911x/sl_wlan_config.h and SiWx917/SiWx917/sl_wlan_config.h are different so this shouldn't get included when building with the efr32